draft-ietf-scitt-architecture.md

```diff
@@ -262,10 +264,9 @@ The Statement is considered opaque to Transparency Service, and MAY be encrypted
 
 Subject:
 
-: a logical collection of Statements about the same Artifact.
-For any step or set of steps in a supply chain there may be multiple statements made about the same Artifact.
-Issuers use Subject to create a coherent sequence of Signed Statements about the same Artifact and Verifiers use the Subject to ensure completeness and Non-equivocation in supply chain evidence by identifying all Transparent Statements linked to the one(s) they are evaluating.
-In SCITT, Subject is a property of the dedicated, protected header attribute `15: CWT_Claims` within the protected header of the COSE envelope.
+: This term has the same definition as in RFC8392.
```

Comment:
```suggestion
: This term has the same definition as in RFC8392, which relies on the definition in RFC7519, which is repeated here with some modifications for readability: The "sub" (subject) claim identifies the principal that is the subject of the CWT.  The claims in a CWT are normally statements about the subject.  The subject value MUST either be scoped to be locally unique in the context of the issuer or be globally unique. The processing of this claim is generally application specific.  The "sub" value is a case-sensitive string containing a StringOrURI value.```

Reply:
It seems essential to repeat this definition based on the discussions we have had regarding identifiers and their duration.

Reply:
done